docs/user_api/index.rst

```diff
@@ -131,6 +140,13 @@ Attributes
    Grid.Mesh2_face_nodes
    Grid.Mesh2_edge_nodes
    Grid.Mesh2_face_edges
+   Grid.polygon_shells
+   Grid.corrected_polygon_shells
+   Grid.corrected_shells_to_original_faces
+   Grid.original_to_corrected_indices
+   Grid.antimeridian_face_indices
```

Comment:
I'd like to keep `polygon_shells` and `corrected_polygon_shells` as part of the user API, since accessing these could be used for user's workflows that require representing the unstructured grid in this format, and not a GeoDataFrame or PolyCollection.

`Grid.antimeridian_face_indices` this similarly should stay as part of the User API, since someone may be interested in knowing which indices into Face Nodes are considered to cross or touch the antemeridian.

`Grid.corrected_shells_to_original_faces` and `Grid.original_to_corrected_indices` we can move to the Internal API, however how would you suggest having internal properties?